zapd/models.py

```diff
@@ -759,3 +823,34 @@ def is_accessible(self):
             self.can_export = True
             return True
 
+class WalletRestrictedModelView(sqla.ModelView):
+    can_create = False
+    can_delete = False
+    can_edit = False
+    #column_list = ['email', 'roles']
+    #column_editable_list = ['roles']
+
+    def is_accessible(self):
+        if not current_user.is_active or not current_user.is_authenticated:
+            return False
+
+        if current_user.has_role('admin'):
+            self.can_export = True
+            return True
+
+class AMDeviceRestrictedModelView(sqla.ModelView):
```

Comment:
dont need this

Reply:
using this as a test at the moment. Want to see what the other side looks like. will disable soon.

Reply:
This is no longer there from the latest update i pulled. Resolving.